src/live-programming.rst

```diff
@@ -36,8 +36,132 @@ Direct Manipulation Programming
 Graphical User Interface Design
 -------------------------------
 
-Game Development
-----------------
+User Interfaces for Game Development
+====================================
+.. :Authors: - Lei Zhang
+
+Overview
+--------
+Starting with Spacewar in 1961, computer games have been around for over four and a half decades.
```

Comment:
Reword to "Computer games have been around for over four and a half decades, starting with Spacewar in 1961."